packages/components/stories/ListComposition/ListComposition.js

```diff
@@ -22,53 +20,27 @@ const titleProps = {
 function CustomList(props) {
 	return (
 		<List.VList id="my-vlist" {...props}>
-			<List.VList.Content label="Id" dataKey="id" width={-1} />
-			<List.VList.Content
-				label="Name"
-				dataKey="name"
-				columnData={titleProps}
-				width={-1}
-				{...CellTitle}
-			/>
-			<List.VList.Content
-				label="Tag"
-				dataKey="tag"
-				columnData={{ selected: true }}
-				width={-1}
-				disableSort
-				{...CellBadge}
-			/>
-			<List.VList.Content label="Description" dataKey="description" width={-1} disableSort />
-			<List.VList.Content label="Author" dataKey="author" width={-1} />
-			<List.VList.Content label="Created" dataKey="created" width={-1} />
-			<List.VList.Content label="Modified" dataKey="modified" width={-1} />
+			<List.VList.Text label="Id" dataKey="id" />
```

Comment:
So sweet http://giphygifs.s3.amazonaws.com/media/hhHcFH0xAduCs/200w_d.gif  ❤️ 
We will do the same on rating / sharing etc so much simpler to use.